backend/app/domains/habits/schemas.py

```
@@ -0,0 +1,78 @@
+"""Habit domain schemas."""
+
+import uuid
+from datetime import date
+from typing import Dict, Any
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)